include/odp/api/spec/packet_io.h

```diff
@@ -159,6 +159,18 @@ typedef struct odp_pktin_queue_param_ovr_t {
 	odp_schedule_group_t group;
 } odp_pktin_queue_param_ovr_t;
 
+/**
+ * Packet vector producer configurations
+ */
+typedef struct odp_pktin_vector_prod_config_t {
+	/** Packet vector pool */
+	odp_pool_t packet_vect_pool;
+	/** Maximum timeout in nanosec to wait for producer form the vector of
+	 * packet event(odp_packet_vect_t).
+	 */
+	uint64_t max_vector_prod_tmo_ns;
+} odp_pktin_vector_prod_config_t;
```

Comment:
I think "prod" part can be dropped from type name: odp_pktin_vector_config_t

Reply:
Ack

Reply:
"vector" can be also removed from config struct field names